spec/util/e2e-test.ts

```diff
@@ -160,6 +166,10 @@ export class IrcBridgeE2ETest {
     }
 
     static async createTestEnv(opts: Opts = {}): Promise<IrcBridgeE2ETest> {
+        const testName = expect.getState().currentTestName?.replace(/[^a-zA-Z]/g, '-');
+        const tracePath = `${traceFilePath}/${testName}.log`;
+        const traceStream = createWriteStream(tracePath, 'utf-8');
```

Comment:
Consider renaming this to `traceLog` for consistency with the `IrcBridgeE2ETest.traceLog`.